overlays/dp1/cnpg-butler-dp1-pooler.yaml

```diff
@@ -0,0 +1,121 @@
+---
+apiVersion: postgresql.cnpg.io/v1
+kind: Pooler
+metadata:
+  annotations:
+  name: usdf-butler-dp1-pooler-transaction-rw
+  namespace: dp1
+spec:
+  cluster:
+    name: usdf-butler-dp1
+  instances: 3
+  pgbouncer:
+    parameters:
+      default_pool_size: "1000"
+      idle_transaction_timeout: "0"
+      log_connections: "1"
+      log_disconnections: "1"
+      max_client_conn: "2000"
+      server_idle_timeout: "0"
```

Comment:
Change to `server_idle_timeout: "60"`.  With Embargo and OR5 testing connections did not clear on their own until this timeout was added.